foundry.toml

```diff
@@ -109,5 +110,6 @@ scroll = { key = "${MAINNET_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io
 sonic = { key = "${MAINNET_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io/v2/api?chainid=146", chain = "146" }
 sei = { key = "${SEI_ETHERSCAN_API_KEY}", url = "https://seitrace.com/pacific-1/api", chain = "1329" }
 taiko = { key = "${MAINNET_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io/v2/api?chainid=167000", chain = "167000" }
+unichain = { key = "${UNICHAIN_ETHERSCAN_API_KEY}", url = "https://unichain.blockscout.com/api?", chain = "130" , verifier = "blockscout" }
```

Comment:
there is a question mark at the end of `url`

Reply:
https://book.getfoundry.sh/reference/forge/forge-verify-contract

This is how foundry describes to use blockscout explorer. I know we have it different for some other networks but in rootstock the ? was missing and it was causing issues so I would suggest that we go with the foundry-proposed way, whenever possible.

Reply:
ok! all clear now